include/Server/src/Server.cpp

```diff
@@ -1,50 +1,59 @@
 #include "Server.h"
 
+#include <sodium.h>
+
 #include <array>
 #include <ctime>
 #include <fstream>
 #include <iostream>
-#include <sodium.h>
-#include "../../ErrorHandler/include/ErrorHandler.h"
 
+#include "../../ErrorHandler/include/ErrorHandler.h"
+#include "config.h"
```

Comment:
Include list needs refactor to meet code style(system libs, third praty libs, our libs)